IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/QuantitySets/Qto_ImpactProtectionDeviceBaseQuantities/DocQuantitySet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocQuantitySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Qto_ImpactProtectionDeviceBaseQuantities" UniqueId="4edbaef5-6f65-4131-a5b8-1772029277a8">
+	<Quantities>
+		<DocQuantity xsi:nil="true" href="Weight_2ueoe8QyrBduWuJHsi21zb" />
```

Comment:
Interesting one. Seems like it needs something more but I cannot put my finger on it. @larswik would like to hear your opinion also.

Reply:
it is a percular one the geometrics possibilities are so varied that anything other than weight could have a vast diference in meaning depending on the particular type of device.

Reply:
How about just having something like nominal dimensions? i.s. bounding box.

Reply:
is that valid within the sense of a quantity? to specify  the bounding box size?

Reply:
Not really, that is why I am tippy-toeing around it. :)